setup.py

```diff
@@ -21,6 +21,7 @@
     author_email='simon@bugsnag.com',
     url='https://bugsnag.com/',
     license='MIT',
+    python_requires='>=3.4.*, <4',
```

Comment:
Shouldn't this be >=3.5?

```suggestion
    python_requires='>=3.5.*, <4',
```

Reply:
yup! looked at it so many times and then missed it.